storage.go

```diff
@@ -3,34 +3,162 @@ package minio
 import (
 	"context"
 	"io"
+	"strings"
 
+	"github.com/minio/minio-go/v7"
+
+	ps "github.com/beyondstorage/go-storage/v4/pairs"
+	"github.com/beyondstorage/go-storage/v4/pkg/iowrap"
+	"github.com/beyondstorage/go-storage/v4/services"
 	. "github.com/beyondstorage/go-storage/v4/types"
 )
 
+const defaultListObjectBufferSize = 100
```

Comment:
Why we set default buffer size 100 here? Any suggestion or reference here? Or just an experienced value?

Reply:
Just an experienced value.